libraries/botbuilder-ai/src/qnaMakerBotComponent.ts

```diff
@@ -7,7 +7,14 @@ import { Configuration, ServiceCollection } from 'botbuilder-dialogs-adaptive-ru
 import { QnAMakerDialog } from './qnaMakerDialog';
 import { QnAMakerRecognizer } from './qnaMakerRecognizer';
 
+/**
+ *
+ */
 export class QnAMakerBotComponent extends BotComponent {
+    /**
+     * @param services Services collection to register dependency injection.
```

Comment:
Dependency injection in JS?

Reply:
Services collection to mimic dependency injection would be ok?

Reply:
Yes, I think that'd be better.